src/tutorials/first_game_ctf.md

```diff
@@ -0,0 +1,273 @@
+2. Drag and drop the **Basic Door** group into the **Damageable Object**.
+3. Select the **Make Children Networked** option when a prompt appears.
+
+![!Damageable Door](../img/FrameworksCTF/FrameworksCTF_DamageableDoor.png){: .center loading="lazy" }
+
+### Customizing the Door
+
+The door will need to be transformed in order to be properly inside of a doorway.
+
+1. Select the **Damageable Object** in the Hierarchy.
+2. Use ++W++ to enable Translation mode and move the door to the doorway.
+3. Use ++E++ to enable Rotation mode and rotate the door if needed.
+4. Navigate to the **Damageable Object > Basic Door > Rotation Root > Geo_StaticContext** folder.
+5. Select the **Whitebox Door 01** object inside of the folder.
+6. Use ++R++ to enable Scale Mode and make door big enough to fit in the doorway.
+
+![!Damageable Door Transform](../img/FrameworksCTF/FrameworksCTF_DamageableDoorTransform.png){: .center loading="lazy" }
+
+### Adding a Damage Effect
+
+Currently, the door will disappear after being destroyed but it looks like it just vanishes. An effect can be added to make it more realistic.
+
+1. In **Core Content**, search for `Damageable Crate`.
+2. Drag and drop the template into the **Hierarchy**.
+3. Select the **Damageable Crate** and open the **Properties** window.
+4. Click the **Find in Asset Catalog** button ![Icon Find](../img/EditorManual/icons/Icon_GenericFind.png) next to the **Destroy on Death Client Template** property. The **Project Content** window should open with the **Crate Damaged Effect** template highlighted.
+   {: .image-inline-text .image-background }
+5. Select the **Damageable Object** in the Hierarchy and open the **Properties** window.
+6. Drag and drop the **Crate Damaged Effect** template into the **Destroy on Death Client Template** property.
+7. Delete the **Damageable Crate** from the Hierarchy.
+
+![!Damageable Door Effect](../img/FrameworksCTF/FrameworksCTF_DamageableDoorEffect.png){: .center loading="lazy" }
+
+### Test the Damageable Door
+
+Play the game and find the door object. Get close to the door and try to open it. Then try shooting it to see the damage effect.
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/FrameworksCTF/FrameworksCTF_DamageableDoorTest.mp4" type="video/mp4" />
+    </video>
+</div>
+
+### Create a New Template
+
+Right click the **Damageable Object** in the Hierarchy and select Create New Template From This. Rename it as `Damageable Door`. Now more doors can be added by dragging them from the **Project Content** window.
+
+![!Damageable Door Template](../img/FrameworksCTF/FrameworksCTF_DamageableDoorTemplate.png){: .center loading="lazy" }
+
+## Learn More
+
+[Publishing Your Game](publishing.md) | [Scenes](scenes.md) | [Damageable Objects](damageable_objects.md) | [Weapons Tutorial](weapons.md)
```

Comment:
Use relative links anytime you link to something in docs. For example, `../references/damageable_objects.md`.